tinyagentos/routes/skill_exec.py

```
@@ -205,6 +205,16 @@ async def _skill_read_layout(args: dict, request: Request) -> dict:
         return {"error": str(exc)}
 
 
+async def _skill_request_decision(args: dict, request: Request) -> dict:
+    """Ask the user a question via the Decisions inbox (human-in-the-loop)."""
+    try:
+        from tinyagentos.tools.decision_tools import execute_request_decision
+
+        return await execute_request_decision(args, request)
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)